src/config.ts

```diff
@@ -34,6 +34,12 @@ const config = {
       clientId: 'ca5c4c520da868387c52',
     },
   },
+  defaults: {
```

Comment:
I don't think we need this.

Each domain may have their own defaults.

Then `PolykeyAgent` and `PolykeyClient` has defaults.

The `bin/options.ts` also has defaults but defaults for the CLI.

So this structure is not needed for now.

Reply:
I thought I had removed that. Fixed.